cli/src/command/wallet.rs

```diff
@@ -44,6 +44,13 @@ pub struct WalletCli {
     pub command: Option<WalletCommand>,
 }
 
+impl WalletCli {
+    pub fn print_help() -> Result<(), Error> {
+        Self::command().bin_name("wallet").print_help()?;
```

Comment:
Not `Wallet:` as in the account?

Reply:
Not because this is the binary, while `Account:` is a prompt